runtime/altair/src/lib.rs

```diff
@@ -856,9 +987,16 @@ construct_runtime!(
 		CrowdloanClaim: pallet_crowdloan_claim::{Pallet, Call, Storage, Event<T>, ValidateUnsigned} = 93,
 		CrowdloanReward: pallet_crowdloan_reward::{Pallet, Call, Storage, Event<T>} = 94,
 		CollatorAllowlist: pallet_collator_allowlist::{Pallet, Call, Storage, Config<T>, Event<T>} = 95,
+		Permissions: pallet_permissions::{Pallet, Call, Storage, Event<T>} = 96,
+		Tokens: pallet_restricted_tokens::{Pallet, Call, Event<T>} = 97,
```

Comment:
Do we want to expose call? Are we restricting this anyways in the filter of system?

Reply:
@offerijns / @mikiquantum ?